appium/lib/screen-object.rb

```diff
@@ -135,7 +135,39 @@ def scroll_up_to_text(text)
   # @param timeout   [Integer] The amount of times in seconds we want to scroll to find the element
   # @return          [Boolean]
   def scroll_text_to_view(text, direction = :down, timeout = 40)
-    wait_until(timeout,'Unable to find element',&->{text_visible?(text, direction)})
+    wait_until(timeout,'Unable to find element', &-> { text_visible?(text, direction) } )
+  end
+
+  # Scrolls in a direction until a element that matches is visible above navigation bar,
+  # @param el      [Element] The text you are looking for on the screen
+  # @param direction [symbol] The direction to search for an string
+  # @param timeout   [Integer] The amount of times in seconds we want to scroll to find the element
+  # @return          [nil]
+  def scroll_rs_element_to_view(el ={}, direction = :down, timeout = 30)
```

Comment:
Why are you adding RS specific method in screen object? It should be totally independent of RS or MMI or any other app that uses it. What happened to the methods `scroll_element_to_view` and `scroll_text_to_view `?